EpsilonAuxBMS/Src/SetContactorsTask.c

```diff
@@ -67,6 +67,18 @@ const uint8_t isDischargeFalsePositive(const uint8_t dischargeSense);
 // Checks if *ptrHasChargeBeenSet is giving a false positive
 const uint8_t isChargeFalsePositive(const uint8_t chargeSense);
 
+// Checks if discharge contactor can be turn on
+const uint8_t canTurnOnDishargeContactor(const uint8_t dischargeSense);
+
+// Checks if charge contactor can be turn on
+const uint8_t canTurnOnChargeContactor(const uint8_t chargeSense);
+
+// Checks if the discharge contactor should be on, but isn't
+const uint8_t isDischargeContactorShouldBeOnButNot(const uint8_t dischargeSense, const uint8_t isdischargeTurningOn);
```

Comment:
Maybe take out the `is` in the name? It's a bit confusing. I was trying to think of better names, but it's sort of a hard function to name haha